src/components/composer/formula_assistant/formula_assistant.ts

```diff
@@ -73,6 +73,10 @@ export class FunctionDescriptionProvider extends Component<Props> {
       this.assistantState.allowCellSelectionBehind = false;
     }, 2000) as unknown as number;
   }
+
+  get formulaArgSeparator() {
+    return this.env.model.getters.getLocale().formulaArgSeparator + " ";
```

Comment:
```suggestion
    return this.env.model.getters.getLocale().formulaArgSeparator + "\u00A0";
```
Not sure if  the unbreakable space was necessary though

Reply:
I didn't put an unbreaking space because it looks like simple spaces are preserved with `t-esc`.